autoload/airline/themes/dracula.vim

```diff
@@ -7,96 +7,84 @@
-let s:gui06 = "#f1fa8c"
-let s:gui07 = "#50fa7b"
-let s:gui08 = "#bd93f9"
-let s:cterm01 = "236"
-let s:cterm02 = "61"
-let s:cterm03 = "215"
-let s:cterm04 = "141"
-let s:cterm05 = "160"
-let s:cterm06 = "228"
-let s:cterm07 = "84"
-let s:cterm08 = "141"
-
-let s:guiWhite = "#f8f8f2"
-let s:guiBlack = "#282a36"
-let s:ctermWhite = "15"
-let s:ctermBlack = "16"
-
-let s:ctermChangedColor = "59"
-let s:guiChangedColor = "#5f5f5f"
-
-" Normal mode
-let s:N1 = [ s:guiBlack , s:gui08 , s:ctermBlack , s:cterm08 ]
-let s:N2 = [ s:guiWhite , s:gui02 , s:ctermWhite , s:cterm02 ]
-let s:N3 = [ s:guiWhite , s:gui01 , s:ctermWhite , s:cterm01 ]
-
-" Insert mode
-let s:I1 = [ s:guiBlack , s:gui07 , s:ctermBlack , s:cterm07 ]
-let s:I2 = [ s:guiWhite , s:gui02 , s:ctermWhite , s:cterm02 ]
-let s:I3 = [ s:guiWhite , s:gui01 , s:ctermWhite , s:cterm01 ]
-
-" Visual mode
-let s:V1 = [ s:guiBlack , s:gui06 , s:ctermBlack , s:cterm06 ]
-let s:V2 = [ s:guiWhite , s:gui02 , s:ctermWhite , s:cterm02 ]
-let s:V3 = [ s:guiWhite , s:gui01 , s:ctermWhite, s:cterm01 ]
-
-" Replace mode
-let s:R1 = [ s:guiBlack , s:gui05 , s:ctermWhite, s:cterm05 ]
-let s:R2 = [ s:guiWhite , s:gui02 , s:ctermWhite, s:cterm02 ]
-let s:R3 = [ s:guiWhite , s:gui01 , s:ctermWhite, s:cterm01 ]
-
-" File changed
-let s:changed = [ s:guiWhite , s:guiChangedColor , s:ctermWhite , s:ctermChangedColor ]
-
-let g:airline#themes#dracula#palette = {}
-let g:airline#themes#dracula#palette.normal = airline#themes#generate_color_map(s:N1, s:N2, s:N3)
-let g:airline#themes#dracula#palette.insert = airline#themes#generate_color_map(s:I1, s:I2, s:I3)
-let g:airline#themes#dracula#palette.visual = airline#themes#generate_color_map(s:V1, s:V2, s:V3)
-let g:airline#themes#dracula#palette.visual = airline#themes#generate_color_map(s:V1, s:V2, s:V3)
-let g:airline#themes#dracula#palette.replace = airline#themes#generate_color_map(s:R1, s:R2, s:R3)
-
-" Inactive mode
-let s:IN1 = [ s:gui04 , s:guiWhite , s:cterm04 , s:ctermWhite ]
-let s:IN2 = [ s:gui04 , s:gui01 , s:cterm04  , s:cterm01 ]
-let s:IA = [ s:IN1[1] , s:IN2[1] , s:IN1[3] , s:IN2[3] , '' ]
-let g:airline#themes#dracula#palette.inactive = airline#themes#generate_color_map(s:IA, s:IA, s:IA)
-
-" Warning info
-let s:WARNING = [ s:guiBlack, s:gui03, s:ctermBlack, s:cterm03 ]
-let s:ERROR = [ s:guiWhite, s:gui05, s:ctermWhite, s:cterm05 ]
-
-let g:airline#themes#dracula#palette.normal.airline_warning = s:WARNING
-let g:airline#themes#dracula#palette.insert.airline_warning = s:WARNING
-let g:airline#themes#dracula#palette.visual.airline_warning = s:WARNING
-let g:airline#themes#dracula#palette.replace.airline_warning = s:WARNING
-
-let g:airline#themes#dracula#palette.normal.airline_error = s:ERROR
-let g:airline#themes#dracula#palette.insert.airline_error = s:ERROR
-let g:airline#themes#dracula#palette.visual.airline_error = s:ERROR
-let g:airline#themes#dracula#palette.replace.airline_error = s:ERROR
-
-" File modified and not saved
```

Comment:
Are we keeping a "file modified and not saved" section in the color scheme ? I couldn't find one below.

Reply:
I removed it because the color that was defined was so closely similar to the regular section that I couldn't really even notice a difference.

Also, on my system, once a file is changed, it switches over to the `_modified` variant permanently, even after it is saved, so it didn't seem very useful. 

Thoughts?

Reply:
Actually, I did notice it because it when from a dark black/purple color to a grey (with the little `[+]`). The grey and the little `[+]` go away after I save.

With the current patch, the middle of the status line is just always the grey modified color.

Reply:
Hm.. maybe my system is different or neovim handles it differently because I definitely didn't notice that.

The color that was used for the "gray" was out of spec. Thoughts on using the `bgdark` color instead if I add it back in?

Reply:
Fine by me.

Reply:
Done

Reply:
Final thought: Would it be useful/helpful to also enable italics for changed filenames if the user has italics enabled?

Reply:
*shrugs* I could go either way on that one, since I don't use them. Is that typical in other IDEs? Your call.

Reply:
That's what they do in VS Code. But I don't particularly find it useful or necessary. I generally know if a buffer has been modified or not if I try to quit out and it doesn't let me.

Reply:
Agreed. Vim let's us know in it's own way. That's one reason I like the color change/indicator. I'm so used to it I don't have to look at it; the corner of my eye sees the change and it's instant feedback.

Reply:
Cool. So we'll keep as-is for the time being then.. Merging!